docs/API/Elements.md

```diff
@@ -114,6 +114,44 @@ var grandParent = this.parent.parent; //This goes up the hierarchy two parents
 
 <br>
 
+You can create vine objects and add them as children, as long as the vine is valid it will accept any argument.
```

Comment:
sentence could be split in two for readability, such as:

"You can create vine objects and add them as children. As long as the vine is valid, it will accept any argument."